updatecli/updatecli.d/updatek3sroot.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: "Update k3sroot version" 
+
+sources:
+ k3sroot:
+   name: Get k3s-root version
+   kind: githubrelease
+   spec:
+     owner: rancher
+     repository: k3s-root
+     token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+     typefilter:
+       release: true
+       draft: false
+       prerelease: false
+     versionfilter:
+       kind: semver
+       # pattern accepts any semver constraint
+       pattern: "*"
+
+targets:
+  dockerfile:
+    name: "Bump to latest k3s-root version in Dockerfile"
+    kind: dockerfile
+    scmid: default
+    sourceid: k3sroot
+    spec:
+      file: "Dockerfile"
+      instruction:
+        keyword: "ARG"
+        matcher: "K3S_ROOT_VERSION"
```

Comment:
I don't remember if this will work out of the box or not, because the value in:
https://github.com/rancher/image-build-flannel/blob/b5f8809a25d349a6ef3ca12d9c4d9ba1584b2285/Dockerfile#L15

has the `"` around. Just commenting, because I really don't remember if the update will happen as expected. If you already tested this locally, please ignore.

Reply:
In the current state, the update will drop the quote around.
I don't think it will be an issue but otherwise we can explicitely add them